### ISSUE TITLE
Feature request: Alert options for when command is done running.

### ISSUE THREAD

**Issue by samuel-massinon** [KristofferC/OhMyREPL.jl#375]
I use [fish's done package](https://github.com/franciscolourenco/done) to notify me when long running commands are done. I feel having something similar in OmMyREPL.jl would be great.

**Comment by KristofferC** [comment]
After #395, OhMyREPL is a thin layer over the native REPL (colorschemes + prompts), so this is out of scope here.
